dts/bindings/dma/renesas,smartbond-dma.yaml

```diff
@@ -19,3 +19,9 @@ properties:
     type: int
     const: 1
     description: Number of block counts supported
+
+# - #dma-cells : Must be <1>.
+# channel: the dma channel, each channel supports one DMA request line
+
+dma-cells:
+  - channel
```

Comment:
Should we also add peripheral trigger id?

Reply:
Added